common/extend-visa.md

```diff
@@ -0,0 +1,39 @@
+# What I need to extend my Visa?
+
+## Rot-Weiß-Rot Karte (Red-White-Red Card)
+**TBD**
+
+## Rot-Weiß-Rot plus Karte (Red-White-Red plus Card)
+To initiate the process of extending the RWR plus card you need to proceed to the [MA 35](https://www.wien.gv.at/kontakte/ma35/) office that covers the district your are currently living. It is recommended that you start this process at least 3 months before your currently Visa date expiration.
+
+You will need to present the follow documentation, to assure that you are in order with their imigration laws:
+
+- KSV1870¹
+- Application Form²
+- Passport photo no older than 6 months
+- Copies of: 
+	Passport
+	Current Visa
+	Rent Contract
+	Last rent payment
+	Work contract (if you are currently working)
+    3 lastest payment slips (yours, if you are currenly working)
+- Meldezettel (if you move since your last application)
+
+ If your RWR card plus it based on your patner Visa, and this person it the money provider, it is import to also have the copies of his/her documents:
+ - KSV1870
+ - Passport
+ - Visa
+ - Work contract
+ - 3 lastest payment slips
+
+After you present the documentation to the MA35 representant, they will scan your finger digitals and hand you a payment bill for the new card that currently costs 160,00.
+
+¹If everything goes well, it takes around 30 to 40 days for new RWR plus card be ready. You need to comeback to the same office with the pay bill receipt and some valid Id documentation (Passaport or currently Visa).
```

Comment:
typos: 
- "for the new", 
- "card to be ready"
- "come back"
- "paid bill"
- "passport"
- "current visa"